README.md

```diff
@@ -1,2 +1,44 @@
-# redhat-marketplace-operators-preprod
-Pre-production catalog for Red Hat Marketplace Operator Bundles
+# Redhat Marketplace operators pre-production catalog
+Pre-production catalog for Red Hat Marketplace Operator Bundles.
+
+# Operator bundle submission
+A new operator bundle submission needs to follow a predefined directory
+structure that is described below in this section. The new submission is
+done through Github pull requests which can be either open by the [CI pipeline][ci-pipeline] or
+manually by a user. The recommended way is to use the CI pipeline because it makes
+sure all necessary data are provided in the correct format. Documentation about how
+to use the pipeline is available in a repo.
```

Comment:
Small thing: `a repo` -> `the pipelines repo`

Reply:
Fixed